### DISCUSSION TITLE
Welcome to HiddenEyeReborn Discussions!

### DISCUSSION THREAD

**Discussion by sTiKyt** [D4rkS3c-Group/HiddenEyeReborn#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by adminpip** [comment]
Can you help me everytime I generate the phishing link when victim opens it it goes to local x remindder page
